### PR TITLE
Release notes for 2.6.0

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

File: public/markdown/changelog.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+## Play 2.6.0
+
+*Released 23 Jun 2016*
+
+* Akka HTTP is now the default server backend. We still will support the Netty server backend but plan to implement most new features on Akka HTTP.
```
Comment (wsargent) [inline public/markdown/changelog.md]: Add "no global application" line

File: public/markdown/changelog.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+* Experimental HTTP/2 support on Akka HTTP.
+* Request attributes for attaching arbitrary typed data to requests
+* Twirl templates now can have a constructor for passing application dependencies. This allows for injectable templates
+* Support for SLF4J markers in both the Java and Scala API, and a special security marker for security-related operations.
+* Configuration improvements in the Scala API, using typeclasses for reading arbitrary types, and removal of the Java configuration API in favor of standard Typesafe Config.
+* Support for compile-time dependency injection in Java, allowing you to create a full Play application in Java without Guice or any dependency injection framework. We have an example using Dagger
```
Comment (wsargent) [inline public/markdown/changelog.md]: We have an example using Dagger _2_

File: public/markdown/changelog.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+* Play WS is now a separate module, and contains several useful improvements including HTTP caching support.
+* Full support for Scala 2.12
+
+[Highlights](https://playframework.com/documentation/2.6.x/Highlights26)
+[Migration Guide](https://playframework.com/documentation/2.6.x/Migration26)
+[Full changelog](hhttps://github.com/playframework/playframework/compare/2.5.x...2.6.0/)
```
Comment (wsargent) [inline public/markdown/changelog.md]: "hhttps"

**Comment by gmethvin** [comment]
updated

**Pull request merged**